plugins/fix_path/__init__.py

```diff
@@ -5,5 +5,5 @@ class Filter:
 
     def file_data_filter(self, file_data):
         filename = file_data['filename']
-        if filename and '//' in filename:
+        if '//' in filename:
```

Comment:
You have not answered the question of why you scan the string twice. As far as I can tell the `if '//' in filename` is completely redundant. Am I missing something?